superset/models/core.py

```diff
@@ -611,10 +611,11 @@ def alter_positions(dashboard, old_to_new_slc_id_dict):
 
         # override the dashboard
         existing_dashboard = None
+        remote_id = dashboard_to_import.params_dict.get("remote_id")
```

Comment:
can we do `remote_id = dashboard_to_import.params_dict.get("remote_id") or dashboard_to_import.id` to be prudent here